src/plugins/ai_assistant_management/selection/public/routes/components/ai_assistant_selection_page.tsx

```diff
@@ -113,6 +114,55 @@ export function AiAssistantSelectionPage() {
             }
           />
         </EuiFlexItem>
+        <EuiFlexItem grow>
+          <EuiCard
+            description={
+              <div>
+                {!securityAIAssistantEnabled ? (
+                  <>
+                    <EuiSpacer size="s" />
+                    <EuiCallOut
+                      iconType="warning"
+                      title={i18n.translate(
+                        'aiAssistantManagementSelection.aiAssistantSelectionPage.thisFeatureIsDisabledCallOutLabel',
+                        {
+                          defaultMessage:
+                            'This feature is disabled. It can be enabled from Spaces > Features.',
+                        }
+                      )}
+                      size="s"
+                    />
+                    <EuiSpacer size="s" />
+                  </>
+                ) : null}
+                <EuiLink
+                  data-test-subj="pluginsAiAssistantSelectionPageDocumentationLink"
+                  external
+                  target="_blank"
+                  href="https://www.elastic.co/guide/en/security/current/obs-ai-assistant.html"
```

Comment:
This should be the up-to-date doc: https://www.elastic.co/guide/en/security/current/security-assistant.html

```suggestion
                  href="https://www.elastic.co/guide/en/security/current/security-assistant.html"
```